app/models/participant_profile.rb

```diff
@@ -2,14 +2,29 @@
 
 class ParticipantProfile < ApplicationRecord
   belongs_to :user
+  # belongs_to :school, optional: true
+  has_many :validation_decisions, class_name: "ProfileValidationDecision"
 
   enum status: {
     active: "active",
     withdrawn: "withdrawn",
   }
 
+  scope :mentors, -> { where(type: Mentor.name) }
+  scope :ects, -> { where(type: ECT.name) }
+  scope :ecf, -> { where(type: [ECT.name, Mentor.name]) }
+
+  scope :npqs, -> { where(type: NPQ.name) }
+
+  scope :sparsity, -> { where(sparsity_uplift: true) }
+  scope :pupil_premium, -> { where(pupil_premium_uplift: true) }
+  scope :uplift, -> { sparsity.or(pupil_premium) }
+
   attr_reader :participant_type
 
+  class_attribute :validation_steps
+  self.validation_steps = []
```

Comment:
Should this be a class method rather than class attribute? Are we planning to dynamically change it?

Reply:
I think I went for a class_attribute to be able to define id declarative in a single line rather than as an override to a class methods (in 3 lines). I don't think there's much reason to choose one over the other, but If you feel strongly about this I am happy to change it to a method...

Reply:
Main reason I find putting it in an attribute weird is that it makes it easily editable (I think), which, well, we probably do not want, especially on a class level.

I don't feel strongly about it though.